locale/en_US/locale.po

```diff
@@ -71,8 +74,47 @@ msgstr "Primary Citation Format"
 msgid "plugins.generic.citationStyleLanguage.settings.citationFormatsPrimaryDescription"
 msgstr "Select which citation format you would like to display by default on your article landing page."
 
+msgid "plugins.generic.citationStyleLanguage.settings.citationFormatsPrimaryDescription.omp"
+msgstr "Select which citation format you would like to display by default on your book landing page."
+
 msgid "plugins.generic.citationStyleLanguage.settings.publisherLocation"
 msgstr "Publisher Location"
 
 msgid "plugins.generic.citationStyleLanguage.settings.publisherLocation.description"
 msgstr "Some citation formats request the geographic location of the publisher, such as \"London, U.K.\"."
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroups"
+msgstr "Author and Translator"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription"
+msgstr "In some cases beside authors there also exists translators. Please choose a role for authors and translators."
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroups.omp"
+msgstr "Author, Editor and Translator"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription.omp"
+msgstr "In some cases beside authors there also exist editors or translators. Please choose a role for authors, editors and translators."
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseAuthor"
+msgstr "Author:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseEditor"
+msgstr "Editor:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseTranslator"
+msgstr "Translator:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseChapterAuthor"
+msgstr "Chapter Author:"
```

Comment:
Remove the `:` from each of these strings and make each one plural:

- Authors
- Editors
- Translators
- Chapter Authors